qutip/tests/test_bloch.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
+import pytest
+
+from qutip.bloch import Bloch
+from qutip import ket, ket2dm
+
+try:
+    import matplotlib.pyplot as plt
+    from matplotlib.testing.decorators import check_figures_equal
+except ImportError:
+    def check_figures_equal(*args, **kw):
+        def _error(*args, **kw):
+            raise RuntimeError("matplotlib is not installed")
+    plt = None
+
+
+check_pngs_equal = check_figures_equal(extensions=["png"])
+
+
+@pytest.mark.skipif(plt is None, reason="matplotlib not installed")
```

Comment:
Probably a good idea. Initially I just followed the behaviour of the only other other test that uses matplotlib (`test_processor`).

I've removed the `skipif` but kept the `try: ... except: ...` around the matplotlib import so that one can at least run other tests without matplotlib installed if one wants to.